src/main/java/net/mcreator/ui/help/HelpLoader.java

```diff
@@ -88,19 +88,18 @@ public static String loadHelpFor(IHelpContext helpContext) {
 				if (getFromCache(helpContext.getEntry()) != null) {
 					helpString.append(renderer.render(parser.parse(getFromCache(helpContext.getEntry()))));
 				} else {
-					helpString.append("No help entry is defined for <tt>").append(helpContext.getEntry())
-							.append("</tt> yet.<br>");
+					helpString.append(L10N.t("help.loader.no_help_entry",helpContext.getEntry(), "</tt> yet.<br>"));
 				}
 
 				if (uri != null && helpContext.getContextName() != null) {
-					helpString.append("<br>Learn more about <a href=\"").append(uri.toString()).append("\">")
-							.append(helpContext.getContextName()).append("</a> on MCreator's wiki.");
+					helpString.append(L10N.t("help.loader.learn_about")).append(uri.toString()).append("\">")

```

Comment:
You still use concatenations, on other places too. Change it so translation entry is something like:

`<br>Learn more about <a href=\"{0}\">{1}</a> on MCreator's wiki.`

Applies to all externalizations